build.sbt

```diff
@@ -104,6 +104,16 @@ val beamSDKIODependencies = Def.settings(
   )
 )
 
+val magnoliaDependencies = Def.settings(
+  libraryDependencies ++= Seq(
+    if (scalaBinaryVersion.value == "2.11") {
```

Comment:
maybe `scalaBinaryVersion.value <= "2.11"` in preparation for scala 2.13? 😅

Reply:
Talked IRL. This branch is for 2.11 only (we no longer support 2.10), and the `else` branch is for >= 2.12.